packages/destination-actions/src/destinations/heap/trackEvent/index.ts

```diff
@@ -98,9 +98,19 @@ const action: ActionDefinition<Settings, Payload> = {
       default: {
         '@path': '$.name'
       }
+    },
+    traits: {
+      label: 'User Properties',
+      type: 'object',
+      description:
+        'An object with key-value properties you want associated with the user. Each property must either be a number or string with fewer than 1024 characters.',
+      default: {
+        '@path': '$.traits'
```

Comment:
Usually you won't find customers adding a traits object at the root of the payload of a page(), screen() or track(). Instead they usually get added in the context object. 

```suggestion
        '@path': '$.context.traits'
```